src/collision/AABB.js

```diff
@@ -60,7 +60,9 @@ AABB.prototype.setFromPoints = function(points, position, angle, skinSize){
     // Compute cosines and sines just once
     var cosAngle = Math.cos(angle),
         sinAngle = Math.sin(angle);
-    for(var i = 1; i<points.length; i++){
+    var len = points.length;
+    
+    for(var i = 1; i < len; i++){
```

Comment:
I was doing a quick search & replace to some similar as the ones i saw in world.step/world.internalstep
to make other things performant too.

I can agree, but if you have things like bodies.length with 1000-10,000 bodies you will notice the difference.

Sidenote: Do you have any jsperf alternatives?
It currently is not so user friendly, no code editor, making changes is quite a pain in the ass & always asks for CAPTCHA.